frontend/src/store/modules/maps.js

```diff
@@ -156,12 +116,8 @@ const actions = {
     commit('setNetwork', network);
   },
 
-  toggleShow2D({ state, commit }) {
```

Comment:
Helper functions like `toggleShow2D` is intended to help clarify the intent and generally make the code easier to read. What is the reason for removing it?

Reply:
I find it reduces the clarity of what the new state will be - ie, I have to understand the "current" state to know what the "new" state will be. Setting the value directly is in this respect clearer, and in terms of code amount the difference is negligible.

Reply:
This was a helper method in the store module and we have now move that logic to: https://github.com/MetabolicAtlas/MetabolicAtlas/blob/3a37a5c7ef1612fd617579659f598e653253babb/frontend/src/components/explorer/MapViewer.vue#L195-L197 and it doesn't really improve the clarity.

Reply:
It would be good to add it back then.